### ISSUE TITLE
'tsconfig.tools.json' template ships unformatted

### ISSUE THREAD

**Issue by peterblazejewicz** [nrwl/nx#954]
_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

# Prerequisites

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

* [x] I am running the latest version
* [x] I checked the documentation and found no answer
* [x] I checked to make sure that this issue has not already been filed
* [x] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

Please describe the behavior you are expecting

When running `npm run format` on brand new, scaffolded project, the generated content should not be changed, including `tsconfig.tools.json`

## Current Behavior

The `npm run format` changes the `tsconfig.tools.json` adding unwanted noise to the output

```diff
--- a/tools/tsconfig.tools.json
+++ b/tools/tsconfig.tools.json
@@ -5,12 +5,7 @@
     "rootDir": ".",
     "module": "commonjs",
     "target": "es5",
-    "types": [
-      "jasmine",
-      "node"
-    ]
+    "types": ["jasmine", "node"]
   },
-  "include": [
-    "**/*.ts"
-  ]
-}
\ No newline at end of file
```

* version of Nx used: 7.1.1
* version of Angular CLI used: 7.0.1

**Comment by peterblazejewicz** [comment]
Seems to be covered by this commit, not yet released?

https://github.com/nrwl/nx/commit/9c10fa7db2777ac1bdb5eb5bb5e119d902bb2bf1#diff-c8acc953e6deb1432e4edaf152d97c4f

**Comment by bcabanes** [comment]
Hi @peterblazejewicz,

The commit you're refering to has been released as part of the "_[Release 7.2.0](https://github.com/nrwl/nx/releases/tag/7.2.0)_" :)

**Comment by peterblazejewicz** [comment]
Thanks! Appreciated. Closing.
